subsys/bluetooth/host/hci_core.c

```diff
@@ -914,6 +914,37 @@ static int hci_le_read_remote_features(struct bt_conn *conn)
 	return 0;
 }
 
+int hci_le_read_remote_version(u16_t handle)
+{
+	int err;
+	struct bt_conn *conn;
+	struct bt_hci_cp_read_remote_version_info *cp;
+	struct net_buf *buf;
+
+	conn = bt_conn_lookup_handle(handle);
```

Comment:
Don't do this - you already have the conn object in conn.c so a bit silly going from bt_conn to handle and then back to bt_conn. I'd just put the helper in conn.c. Not everything HCI related has to be in hci_conn.c. Just the main/core things (hence the name of the file).

Reply:
But the conn in `bt_conn_get_info` is const, and I got error that the const identifier was being discarded.

Reply:
What error exactly?

Reply:
I updated such that I have newest version of zephyr and now I get another error:
```
In file included from C:/ncs/zephyr/include/arch/arm/cortex_m/cmsis.h:17:0,
                 from C:/ncs/zephyr/include/arch/arm/cortex_m/mpu/arm_mpu_v7m.h:10,
                 from C:/ncs/zephyr/include/arch/arm/cortex_m/mpu/arm_mpu.h:13,
                 from C:/ncs/zephyr/include/arch/arm/arch.h:246,
                 from C:/ncs/zephyr/include/arch/cpu.h:17,
                 from C:/ncs/zephyr/include/kernel_includes.h:34,
                 from C:/ncs/zephyr/include/kernel.h:17,
                 from C:/ncs/zephyr/kernel/include/kernel_structs.h:10,
                 from C:/ncs/zephyr/arch/arm/core/offsets/offsets.c:26:
c:/ncs/zephyr/soc/arm/nordic_nrf/nrf52/soc.h:17:10: fatal error: nrf.h: No such file or directory
 #include <nrf.h>
```

Reply:
@Blackladder You need to do 'west update'